oas

```diff
@@ -1,339 +1,56 @@
-                            "type": "object",
-                            "required": [
-                              "type",
-                              "coordinates"
-                            ],
-                            "properties": {
-                              "type": {
-                                "pattern": "^Point$",
-                                "type": "string"
-                              },
-                              "coordinates": {
-                                "description": "See https://json.schemastore.org/geojson.json"
-                              }
-                            },
-                            "example": {
-                              "type": "Point",
-                              "coordinates": [
-                                12.554,
-                                43.43234,
-                                2
-                              ]
-                            }
-                          }
-                        },
-                        "additionalProperties": {
-                          "$ref": "#/components/schemas/Parameters"
-                        }
-                      }
-                    }
-                  }
-                }
-              }
-            }
-          },
-          "400": {
-            "description": "Invalid or missing parameters. Also returned when fullfilling the request would transfer too much data (up to the implementor). The content SHOULD specify the reason as a json string.",
-            "content": {
-              "application/json": {
-                "schema": {
-                  "$ref": "#/components/schemas/Error"
-                }
-              }
-            }
-          },
-          "401": {
-            "description": "Invalid or missing credentials",
-            "content": {
-              "application/json": {
-                "schema": {
-                  "$ref": "#/components/schemas/Error"
-                }
-              }
-            }
-          },
-          "404": {
-            "description": "No accessible sensors have ever been placed within the Boundary.",
-            "content": {
-              "application/json": {
-                "schema": {
-                  "$ref": "#/components/schemas/Error"
-                }
-              }
-            }
+        "summary": "Get tracking data for a returnable asset e.g. pallets",
+        "description": "Get tracking data for a returnable asset e.g. pallets with specified GSIN and SSCC GS1 specification uids",
+        "operationId": "get_asset_data_data_get",
+        "parameters": [
+          {
+            "required": true,
+            "schema": {
+              "title": "Gsin",
+              "type": "string"
+            },
+            "name": "GSIN",
```

Comment:
Provide description. Also explain in doc.html
Are both GSIN and SSCC required or just one of the two?
Would it make sense to include one of the two in the url path like: GET /tracking_data/<asset_identifier>?